src/utils/join.js

```
@@ -65,29 +72,60 @@ function normalizeString(path, aar) {
+}
+
 function normalize(path) {
   if (!path.length) return '.'
 
-  const isAbsolute = path[0] === '/'
+  // Normalise backslashes to forward slashes before any other processing.
+  path = path.replace(/\\/g, '/')
+
+  const drivePrefix = getWindowsDrivePrefix(path)
+  // isAbsolute: Unix root ('/foo') OR Windows drive+slash ('C:/foo').
+  const isAbsolute = path[0] === '/' || (drivePrefix !== null && path[2] === '/')
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)